aio_pika/tools.py

```diff
@@ -37,6 +37,11 @@ def iscoroutinepartial(fn: Callable[..., Any]) -> bool:
     return asyncio.iscoroutinefunction(parent)
 
 
+def _task_done(future: asyncio.Future) -> None:
+    exc = future.exception()
```

Comment:
This will raise `asyncio.CancelledError` in case of the `future` was cancelled. @mosquito if it's ok then we can merge this.

Reply:
It's should be raised in event loop stack and might be handled by `loop.default_exception_handler` in base case